payment_paypal/indico_payment_paypal/controllers.py

```diff
@@ -5,21 +5,24 @@
 # them and/or modify them under the terms of the MIT License;
 # see the LICENSE file for more details.
 
+import re
+import time
 from itertools import chain
 
 import requests
-from flask import flash, redirect, request
 from flask_pluginengine import current_plugin
-from werkzeug.exceptions import BadRequest
 
+from indico.core.db import db
 from indico.modules.events.payment.models.transactions import TransactionAction
 from indico.modules.events.payment.notifications import notify_amount_inconsistency
 from indico.modules.events.payment.util import register_transaction
 from indico.modules.events.registration.models.registrations import Registration
 from indico.web.flask.util import url_for
 from indico.web.rh import RH
 
+from flask import flash, redirect, request
 from indico_payment_paypal import _
+from werkzeug.exceptions import BadRequest
```

Comment:
this one is still out of order, it should be in the second import group (make sure to run isort before right commiting)

Reply:
I ran isort before every commit...